target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -33,6 +33,11 @@ netgear,r7800)
 	ucidef_set_led_netdev "wan" "WAN" "${boardname}:white:wan" "eth0"
 	ucidef_set_led_ide "esata" "eSATA" "${boardname}:white:esata"
 	;;
+qxwlan,e5200)
+	ucidef_set_led_usbport "usb1" "USB 1" "${boardname}:green:usb1" "usb1-port1" "usb2-port1"
```

Comment:
Could you please use the device-tree based usbport led trigger setup, I remember mentioning this in your previous
PR for your E2600AC C1/C2 https://github.com/openwrt/openwrt/pull/1655#discussion_r245761242 .